docs/explanation/lifecycle-layer.rst

```diff
@@ -0,0 +1,198 @@
+
+From lifecycle to OCI layer
+===========================
+
+Rockcraft is a tool that creates OCI images using the same concepts and
+mechanisms that create snaps and charms: the lifecycle language from
+craft-parts. There is a significant difference between the works of the Craft
+lifecycle and the OCI specification, and one of Rockcraft's jobs is to bridge
+the gap between these two worlds. This page describes how this is accomplished.
+
+It is not necessary to know these details to use the tool effectively, but they
+might illuminate some concepts and help understand *why* the contents of a given
+rock are the way they are.
+
+Consider the following snippet of a ``rockcraft.yaml`` that creates a rock
+containing a bare-bones Python 3.10 interpreter:
+
+.. code-block:: yaml
+
+   # (...)
+   base: ubuntu@22.04
+
+   parts:
+     python-part:
+       plugin: nil
+       stage-packages:
+         - python3-minimal
+
+This rock has Ubuntu Jammy as its base, and includes ``python3-minimal``.
```

Comment:
Always refer to Ubuntu by the release name and not code name, this is Ubuntu 22.04